### DISCUSSION TITLE
the readme says go to newwatch.slingbox.com to get pw but its down now what to do thx

### DISCUSSION THREAD

**Discussion by kocoman2** [GerryDazoo/Slinger#370] [General]
the readme says go to newwatch.slingbox.com to get pw but its down now what to do thx

**Comment by womeara6** [comment]
I was not aware of this project and just tried to use my slingbox only to find that its been deactivated.  I didn't think this would happen until 2023.  I too missed getting the password information before the servers were taken down.  Hopefully Gerry has a solution.

**Reply by socelillyhammer** [reply]
see #374 and also #381 you can still get your passwords

**Comment by abarberenaCPDS** [comment]
hey @kocoman2 👋, yeah, to confirm, `newwatch.slingbox.com` was available until the cutoff. but keep an eye in the repo, hopefully an alternative way to retrieve will be available soon. 🤞

**Reply by socelillyhammer** [reply]
see #374 and also #381

**Comment by socelillyhammer** [comment]
Ok,  all of you new guys are in luck. It also works on the 350 and it should work for the M1 and M2

Check out my post #374 

If you can still login you are in luck

I only ask for one thing, please help me out with getting the verizon fios one box new remote to work.  The current universal remote codes do not work on the new verizon fios boxes.

**Reply by Toei79** [reply]
> Ok, all of you new guys are in luck. It also works on the 350 and it should work for the M1 and M2
> 
> Check out my post #374
> 
> If you can still login you are in luck
> 
> I only ask for one thing, please help me out with getting the verizon fios one box new remote to work. The current universal remote codes do not work on the new verizon fios boxes.
how you switch inputs?
i fix mine it was the password.

**Reply by Toei79** [reply]
i figure out, video source gives the one to use.
i was confuse with remote , but remote its for use with tv cable boxes.

**Reply by socelillyhammer** [reply]
@Toei79 im sorry I dont understand

**Reply by Toei79** [reply]
> @Toei79 im sorry I dont understand

i wanted switch component , and composite,  then i figure out its change  numbers in the .ini file
0 composite, 1 component.

**Reply by socelillyhammer** [reply]
@Toei79 yes correct.  glad you figured it out.  and is your remote working now too?

**Reply by Toei79** [reply]
no, actually i use iptv tv.  but i have plans to use this , its not such urgent like you guys using tv boxes.
at minimum you can stream something there.  like a dvd , blu ray full of episodes something like that. just thinking.
